OverlayPlugin.Core/EventSources/FFXIVOptionalEventSource.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+using Advanced_Combat_Tracker;
+using Newtonsoft.Json.Linq;
+
+namespace RainbowMage.OverlayPlugin.EventSources
+{
+    /**
+     * This EventSource contains lines that should work without FFXIV, but add additional information when FFXIV is present
```

Comment:
Personally speaking, I feel like these things are all ffxiv-specific in their data still.  What non-ffxiv things are going to have log lines in this format with this data?  Am I misunderstanding?

My initial inclination would be to only have FFXIVEventSource and not the optional/required split.  I don't feel strongly, I just don't believe this potential use case merits the complication.

Reply:
There was an additional comment in this file below at line 165 that was basically `// @TODO: DispatchEvent for LogLineEvent should probably always be emitted since there's nothing FFXIV-specific for this event?`, but I guess that got lost at some point.

There was some additional language there indicating that it might be good to offer a way to extend the event handler per game in case some other game wanted to extend for their own support structure, or to maybe make the separator character customizable.

Reply:
I think that a future followup PR could eliminate this file entirely by replacing it with some sort of RegisterLogLineHandler function which can handle the split and dispatch its own events, and then we could have an FFXIV log line listener register to that and handle the format for FFXIV log lines?